### DISCUSSION TITLE
Is there a HOWTO for someone who has never actually used KDE Connect?

### DISCUSSION THREAD

**Discussion by bornagainpenguin** [andyholmes/valent#380] [Help]
It seems that there is an assumption that anyone using this is already familiar with the KDE Plasma version.  That's not the case for me.  I have the application installed in Linux but it doesn't seem to see the phone.  Do I need a helper app on the Android side?  Does the phone need to be tethered to work?  If so is there a specific mode it needs to be in?

Sorry for the many questions but I'm really coming into this from scratch.  Would appreciate some basic handholding on how to get things started.  Thanks for your time.

**Comment by andyholmes** [comment]
Unfortunately there's no documentation for Valent at this point, since too many things are still changing to offer accurate documentation.

I can recommend the [official KDE Connect documentation](https://community.kde.org/KDEConnect) or [GSConnect's guide](https://github.com/GSConnect/gnome-shell-extension-gsconnect/wiki/Help#connecting-an-android-device), which should be relevant.

You may want to stick with [KDE Connect](https://kdeconnect.kde.org) or [GSConnect](https://github.com/GSConnect/gnome-shell-extension-gsconnect) if you're looking for something that just works though, since this project is still a bit unstable.

**Reply by bornagainpenguin** [reply]
Personally I'm looking for something that runs well on Mate desktop.  I've ignored suggestions to use KDE Connect over the past few years because I'd prefer something that fits in better with Mate and because for some unknown reason I really do not like KDE.  I've tried it many times over the past two decades and it just doesn't do it for me.

As for documentation, I was so clueless to begin with that I didn't even realize that a helper app was required to be installed on the phone.  I was eventually able to discover that and try out the program.  It worked initially and then seemed to stop.  I will be happily checking in off and on now that I know of the program's existence.  I just wonder how many other people may have never used even the KDE Connect original version and would appreciate a quick start guide that at least mentions the need for a helper app to be installed on the target device?

Thank you for taking the time to respond and for making this application!

**Reply by andyholmes** [reply]
>  I just wonder how many other people may have never used even the KDE Connect original version and would appreciate a quick start guide that at least mentions the need for a helper app to be installed on the target device?

Good user help documentation is planned, including a first-start wizard, but at the moment things are still just changing too fast. If I put up screenshots or directions, they could very well be inaccurate tomorrow.

> Thank you for taking the time to respond and for making this application!

No problem! I do appreciate the reminder that not everyone has heard of or used a KDE Connect client before.

The last project had a wiki, but it was hard to keep up with and never translated for other languages. I'd like to have in-app troubleshooting that's actually useful this time around :)
